spec/acceptance/iis_application_pool_spec.rb

```diff
@@ -22,6 +22,17 @@
       it_behaves_like 'an idempotent resource'
     end
   end
+  
+  context "when setting props" do
+    it 'sets the state to stopped' do
+      apply_manifest(%{
+        iis_application_pool { 'test':
+          ensure => present,
+          state => 'stopped'
+        }
+      }, acceptable_exit_codes: (0...256))
```

Comment:
Yep, I know, I was trying to have some test for the new properties being set, but still figuring out this syntax.